### DISCUSSION TITLE
Necesito que me envien al correo la contraseña que pague para usar este programa

### DISCUSSION THREAD

**Discussion by Fitzroy507** [aclap-dev/video-downloadhelper#1377] [License & Purchase requests]
### What's your issue?

Saludos, pague por el servicio pero no logro encontrar el correo con la contraseña.

Estoy convencido que en sus registros debe aparecer ya que en mi banco aparece el pago.

Me mantengo atento a su ayuda.

Gracias

### The 10 first digits (only 10) of your license number or email address

fitzroyhay@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, I sent you an email.
